src/documentation/0000-node-fs-fd-use-case-write-method/article.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Use case of file descriptor returned by fs.open() method in fs.write()
+description: "Use of write method in fs module to handle writing huge amount of data in chunks."
+authors: TanmayBanerjee-23rd
+section: Quick Start
+category: learn
+---
+
+# Introduction to fs module write method.
+
+Using `fs.write()` method we can write a file in a more controlled manner in comparision to `fs.writeFile()` or `fs.writeFileSync()`.
+
+## Working with fs.write() method.
+
+Since `fs.writeFile()` and `fs.writeFileSync()` writes the entire content of data in memory before returning, the methods will make sure the entire data is written in a go which in case of huge sized data will have a great performance impact in terms of memory usage and speed of execution of the program.
+
+So, we can manage the above situation by using `fs.write()` method which gives us more control over writing the file. How? We will see it below.
+
+`fs.write()` Asynchronously writes data to the file referenced by the supplied file descriptor returned by `fs.open()`. Since it's asynchronous in nature which will help not to block our call stack.
```

Comment:
```suggestion
`fs.write()` asynchronously writes data to the file referenced by the supplied file descriptor returned by `fs.open()`. Since it's asynchronous in nature which will help not to block our call stack.
```